### PR TITLE
Mafia updates

### PR DESCRIPTION
**Pull request by bumbadadabum**
also apparently a statcalc fix I didn't make a PR for... oops

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1750,8 +1750,8 @@ exports.commands = {
 		"Adding the parameter 'all' or 'table' will display the information with a table of all type combinations."],
 
 	statcalc: function (target, room, user) {
-		if (!this.canBroadcast()) return;
 		if (!target) return this.parse("/help statcalc");
+		if (!this.canBroadcast()) return;
```
Comment (panpawn) [inline chat-plugins/info.js]: What change is this...?

File: chat-plugins/mafia-data.js [hunk 2]
```diff
@@ -6,9 +6,17 @@
...
 	copReport: function (target) {
-		if (target.class.side === 'town') {
+		let side = null;
+
+		if (target.class.appearAs) {
+			side = target.class.appearAs;
+		} else {
+			side = target.class.side;
+		}
+
```
Comment (Slayer95) [inline chat-plugins/mafia-data.js]: ``let side = target.class.appearAs || target.class.side``

File: chat-plugins/info.js (hunk 1 above)
Comment (Slayer95) [inline chat-plugins/info.js]: It's fine. It suppresses the announcement of the command usage for ``!statcalc``

Comment (panpawn) [inline chat-plugins/info.js]: Oh, I didn't see the change in the git, now I see that it was moved to another line, ops

File: chat-plugins/mafia-data.js (hunk 2 above)
Comment (bumbadadabum) [inline chat-plugins/mafia-data.js]: Ah right that works in JS

**Comment by bumbadadabum** [comment]
fixed that

**Pull request merged**